EMS/common-bundle/composer.json

```diff
@@ -37,7 +37,7 @@
 		"symfony/console": "^5.4",
 		"symfony/dom-crawler": "^5.4",
 		"symfony/event-dispatcher": "^5.4",
-		"symfony/expression-language": "^5.4",
```

Comment:
Does it retro compatible?

Reply:
Yes works the same as before 
Tested the book evaluate function and the date and date modify